lib/pocketape.dart

```diff
@@ -23,15 +25,20 @@ abstract class Pocketape {
           onDone: controller.close,
         );
         if (_count == 1) {
+          print("Start");
+          if (Platform.isAndroid && !(await _requestPermission())) {
+            controller.addError(CameraPermissionDeniedException("Camera permission denied"));
+            return;
+          }
           await _platformChannel.invokeMethod('startMeasure');
         }
       },
       onCancel: () async {
         _count--;
-        subscription.cancel();
         if (_count == 0) {
           await _platformChannel.invokeMethod('stopMeasure');
         }
+        await subscription.cancel();
```

Comment:
I suppose this is done to resolve the async coordinates delay (not starting at 0,0)?

I do not really understand why this improves the situation. The only thing I can think of is that there are still events added to the stream from android which are buffered. Thus if we unsubscribe too early they are kept in the buffer and sent on the next subscription (trace() call)

Reply:
Yes, that's what I think.